extensions/healthie/actions/getMostRecentMetricEntry/config/dataPoints.ts

```diff
@@ -0,0 +1,8 @@
+import { type DataPointDefinition } from '@awell-health/extensions-core'
+
+export const dataPoints = {
+  lastMetricValue: {
+    key: 'lastMetricValue',
+    valueType: 'json',
+  },
```

Comment:
Now that I see this, I think it's best we create three separate data points:
1. metricId (string)
2. lastMetricValue (number)
3. createdAt (date) --> make sure it's stored as an ISO8601 date as I see that Healthie returns dates in this format: 2023-10-10 12:20:53 +0200